mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-get-parties-7.2.0.plantuml

```diff
@@ -74,149 +75,133 @@ end box
 ' START OF FLOW
 
 group Get Party Details
-    PAYER_FSP --> ALS_API: Request to get parties's FSP details \nGET - /parties/{TYPE}/{ID}?currency={CURRENCY}. \n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200zx, 300x, 310x, 320x</color>
+    PAYER_FSP ->> ALS_API: Request to get parties's FSP details\nGET - /parties/{TYPE}/{ID}?currency={CURRENCY}\n<color #388724><b>Response code:</b> 202</color>\n<color #red><b>Error code:</b> 200zx, 300x, 310x, 320x</color>
```

Comment:
We use '-->' to denote an async callback response in our other sequence diagrams do we not?